tfs/src/main/java/hudson/plugins/tfs/JenkinsEventNotifier.java

```diff
@@ -74,6 +79,17 @@ public static String getApiJson(final String url) {
             }
             final String rootUrl = jenkins.getRootUrl();
             final String fullUrl = urlCombine(rootUrl, url, "api", "json");
+            SSLContext sslContext = SSLContexts.custom()
+                    .useTLS()
+                    .build();
+
+            SSLConnectionSocketFactory f = new SSLConnectionSocketFactory(
+                    sslContext,
+                    new String[]{"TLSv1.1", "TLSv1.2"},
```

Comment:
Are there constants for TLSv1.1 and 1.2 strings?

Reply:
There aren't. I didn't add any as these values aren't used elsewhere. That being said, it could certainly be added.